src/NHSD.BuyingCatalogue.Organisations.Api/Models/ApplicationUser.cs

```diff
@@ -9,6 +9,8 @@ public sealed class ApplicationUser : IdentityUser
 
         public string LastName { get; set; }
 
+        public string DisplayName => FirstName + " " + LastName;
```

Comment:
Just a suggestion as it boils down to personal preference. Have you considered using string interpolation in this case?

Reply:
I could've done but chose not to.

`public string DisplayName => $"{FirstName} {LastName}";`

Maybe the string interpolation looks slightly neater... I'm not sure. The string concatenation version just seems to read ever so slightly better to me. However, I'm easy on this one.

Reply:
I prefer the $ one, for sure

Reply:
That's two votes for the string interpolation option so I'll switch it to that.